kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportConfig.kt

```diff
@@ -0,0 +1,1225 @@
+ * - What coverage bounds are enforced by current rules
+ * - What kind of bounds (branches, lines, bytecode instructions) are checked by bound rules.
+ */
+public interface KoverVerifyRule {
+    /**
+     * Specifies by which entity the code for separate coverage evaluation will be grouped.
+     * [GroupingEntityType.APPLICATION] is used if value is empty.
+     *
+     * Empty by default.
+     */
+    public val groupBy: Property<GroupingEntityType>
+
+    /**
+     * Specifies that the rule is checked during verification.
+     *
+     * The rule will be checked if the value is `false` or empty.
+     *
+     * Empty by default.
+     */
+    public val disabled: Property<Boolean>
+
+    /**
+     * Specifies that the rule is checked during verification.
+     */
+    @Deprecated(
+        message = "Property was renamed to disabled and inverted. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        replaceWith = ReplaceWith("disabled"),
+        level = DeprecationLevel.ERROR
+    )
+    public var isEnabled: Boolean
+        get() {
+            throw KoverDeprecationException("Property isEnabled was renamed to disabled and inverted. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+        }
+        set(value) {
+            throw KoverDeprecationException("Property isEnabled was renamed to disabled and inverted. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+        }
+
+    @Deprecated(
+        message = "Property was renamed to groupBy. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        replaceWith = ReplaceWith("groupBy"),
+        level = DeprecationLevel.ERROR
+    )
+    public var entity: GroupingEntityType
+        get() {
+            throw KoverDeprecationException("Property entity was renamed to groupBy. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+        }
+        set(value) {
+            throw KoverDeprecationException("Property entity was renamed to groupBy. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+        }
+
+    @Deprecated(
+        message = "It is forbidden to override filters for a specific report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        level = DeprecationLevel.ERROR
+    )
+    public fun filters(config: Action<KoverReportFiltersConfig>) {
+        throw KoverDeprecationException("It is forbidden to override filters for the verification report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
```

Comment:
Btw, the migration guide does not say anything about removing report-specific filters. I suggest you add section with `The concept of format-specific reports filters has been removed` right after `More about the reports variant`